Remove graph_jobs management command and asciichartpy dependency

TheRealHaoLiu opened the pull request:
##### SUMMARY
Remove unused Django management command `graph_jobs` and its dependency. The `graph_jobs` command was used for plotting job status over time on the terminal but is no longer needed. This cleanup removes the command file and the `asciichartpy` dependency that was only used by this command.

##### ISSUE TYPE
- Bug, Docs Fix or other nominal change

##### COMPONENT NAME
- CLI

##### ADDITIONAL INFORMATION
Files removed:
- `awx/main/management/commands/graph_jobs.py` - Django management command for plotting job status
- `asciichartpy` dependency from `requirements/requirements.in`

The `asciichartpy` package was only used by the `graph_jobs` command, so it's safe to remove from the requirements. This change reduces the dependency footprint without affecting any other functionality.

```
# Before: graph_jobs command existed
python manage.py graph_jobs --help

# After: command no longer exists, dependency removed
```

<!-- CURSOR_SUMMARY -->
---

> [!NOTE]
> Removes an unused CLI command and its sole dependency to reduce footprint.
> 
> - Deletes `awx/main/management/commands/graph_jobs.py`
> - Drops `asciichartpy` from `requirements/requirements.in` and `requirements/requirements.txt`; removes `licenses/asciichartpy.txt`
> - Cleans up related note in `requirements.txt` referencing `setuptools` via `asciichartpy`
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit a06f2da919ceed6e73f491f48b528be056180b2e. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>
<!-- /CURSOR_SUMMARY -->

TheRealHaoLiu committed:
Remove graph_jobs management command and asciichartpy dependency

Deleted the awx/main/management/commands/graph_jobs.py file and removed the asciichartpy package from requirements. This cleans up unused code and dependencies related to terminal job status graphing.

AlanCoding reviewed (dismissed):
still need to remove the license

TheRealHaoLiu committed:
Remove asciichartpy license file

Deleted the asciichartpy.txt license file from the licenses directory. This may indicate the package is no longer used or the license documentation has been relocated.
